test/very_much_not_secure/jwks.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
As the name implies this should never be used in production! It is only for [testing](https://github.com/alexjpaz/prestissimo/blob/c669c81730c8cc78e2981de9c7f1346e106c5599/src/http/authorizer.spec.js) the `authorization` class.